### ISSUE TITLE
File created using tmp.fileSync() cannot be deleted on Windows

### ISSUE THREAD

**Issue by yoavain-sundaysky** [raszi/node-tmp#202]
## Operating System
- [X] Windows 10

## NodeJS Version
- [X] 12.x

## Tmp Version
0.1.0

## Expected Behavior
the following code should delete temp file:
```
import * as tmp from "tmp";

let tempFile = tmp.fileSync();
fs.unlinkSync(filename);
```

## Experienced Behavior
File stays even after node exits.

Note:

1. When creating file using fs.writeFileSync(), i'm able to delete it.
2. Tried with fs-extra remove() and the result is the same
3. When creating temp directory using tmp.dirSync() it is possible to delete it using fs-extra remove()

**Comment by silkentrance** [comment]
This might be related to #198 and #199.

**Comment by silkentrance** [comment]
Fixed with #198.
